doc/troubleshooting.md

```diff
@@ -185,8 +185,8 @@ Re-enable the CRC check:
 
 No more alarms should be raised and MotoROS2 should now be installed.
 
-If the alarm is raised again, save a copy of the `PANELBOX.LOG` from the robot's teach pendant.
-Open a new issue on the [Issue tracker](https://github.com/yaskawa-global/motoros2/issues), describe the problem and attach the `PANELBOX.LOG` to the issue.
+If the alarm is raised again, save a copy of the `ALL.PRM` from the robot's teach pendant.
+Open a new issue on the [Issue tracker](https://github.com/yaskawa-global/motoros2/issues), describe the problem and attach the `ALL.PRM` to the issue (alternatively: send it to the MotoROS2 developers in a private email).
```

Comment:
> (alternatively: send it to the MotoROS2 developers in a private email).

I'm curious what the reason for this is? There isn't any proprietary in a parameter file.

Reply:
I just wanted to pre-empt potential questions about this. I've had many such 'private emails' sent to me just because a robot was being used in a 'sensitive context' (or commercial) and *everything* was considered either proprietary or confidential.